babel.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   presets: [
     '@vue/cli-plugin-babel/preset'
-  ]
+  ],
+  plugins: ['@babel/plugin-proposal-private-methods']
```

Comment:
Should we do this in Create and Filings UIs, too?

Reply:
I had to do this to make the new PDF stuff work

Reply:
@JazzarKarim - FYI, when you (or I) update Filings UI packages.

Reply:
Travis, can you have a peek if Create UI needs this, please?

Reply:
I would include it in Create UI

Reply:
I created ticket 16462 for this. Resolving THIS conversation.